sources/packages/api/src/database/entities/institution.model.ts

```diff
@@ -78,11 +72,5 @@ export class Institution extends RecordDataModel {
   })
   institutionAddress: InstitutionAddress;
 
-  @ManyToMany(() => User, { eager: false, cascade: true })
```

Comment:
Can you elaborate a bit on why this relationship was removed?

Reply:
Currently-
1. There is Institution user entity  
2. Which connects institution, user, + location (optional) + auth type_role